Multisig/Data/Services/Safe Client Gateway Service/SCGModels.swift

```diff
@@ -0,0 +1,380 @@
+//
+//  SCGModels.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 13.08.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct SCGPage<T: Decodable>: Decodable {
+    let next: String?
+    let previous: String?
+    let results: [T]
+}
+
+struct SCGTransactionSummary: Decodable {
```

Comment:
As ClientGateway will be our default backend, I would remove the prefix ClientGateway& SCG

Reply:
I will keep it for  the types  that  collide with existing  types